src/color.rs

```diff
@@ -26,9 +26,10 @@ where
     }
 }
 
+/// Serialize the alpha copmonent of a color according to the specification.
 /// <https://drafts.csswg.org/css-color-4/#serializing-alpha-values>
 #[inline]
-fn serialize_alpha(
+pub fn serialize_alpha(
```

Comment:
Let's call this serialize_color_alpha or something then, since there's a other alpha values like opacity that are just a plain number / percent.